quiche/src/lib.rs

```diff
@@ -4805,6 +4804,37 @@ impl Connection {
         MIN_CLIENT_INITIAL_LEN
     }
 
+    /// Queue a PING frame to be sent for the active path if no other
+    /// ACK-eliciting frame would otherwise be sent. See
+    /// [`send_ack_eliciting_on_path()`] for more detail.
+    ///
+    /// [`send_ack_eliciting_on_path()`]:
+    /// struct.Connection.html#method.send_ack_eliciting_on_path
+    pub fn send_ack_eliciting(&mut self) -> Result<()> {
+        if self.is_closed() || self.is_draining() {
+            return Ok(());
+        }
+        self.paths.get_active_mut()?.needs_ack_eliciting = true;
+        Ok(())
+    }
+
+    /// Queue a PING frame to be sent for the specified path if no other
+    /// ACK-eliciting frame would otherwise be sent. Calling this method
+    /// multiple times before calling send will have no effect.
```

Comment:
My comment from above applies here too. However, this function can return an error if the user-provided path doesn't exist, so maybe we need to document that possibility in the API a little more explicitly.